### ISSUE TITLE
3D model issues above and between the wings

### ISSUE THREAD

**Issue by legoboyvdlp** [c172p-team/c172p-detailed#1062]
![image](https://user-images.githubusercontent.com/10434911/38826237-1cde7dd2-41a7-11e8-9b78-383a67625f90.png)

Two rivets overlapping and a nasty flickering thing (half-circled in black) on the top of the fuselage. I'd imagine it is the cockpit roof showing outside?

There is also a rather obvious  seam here:
![image](https://user-images.githubusercontent.com/10434911/38826330-5c6433f2-41a7-11e8-9396-bccb08bcebf9.png)

But that is not a major issue, as I had to specifically look for 3D problems to spot it.

The rivets around the fuel caps, however do have some problems:
![image](https://user-images.githubusercontent.com/10434911/38826364-73f8af16-41a7-11e8-87b0-065754690481.png)

I appreciate that it is a lot of work to do these rivets, so please don't feel you have to do them right now, this can wait a bit. But the first issue definitely should be fixed.

**Comment by gilbertohasnofb** [comment]
@legoboyvdlp I think the issue is that the wing is broken into several different pieces now and mapping them has not been too easy. Normally the best approach to modelling is to first create a 3D model and then to UV-map and texture it, but we need to do the opposite: adjust textures to a modified model. Anyway, perhaps @wlbragg will be interested in taking a look at these, but I am afraid it will be quite difficult to have all rivets looking perfect given our model.

@wlbragg about the rivets around the fuel cap, I thought we had fixed those, did some improvements get lost?

**Comment by legoboyvdlp** [comment]
Perhaps my branch did not actually have that fix - if so, apologies.

**Comment by wlbragg** [comment]
@legoboyvdlp  I think your in a branch that doesn't have the latest revisions. Here is what the most current work looks like.

![riv](https://user-images.githubusercontent.com/11069680/38839921-8043191e-41a2-11e8-92e2-5fdd1ab448b3.jpg)

**Comment by legoboyvdlp** [comment]
The rivets on the left seem pinched slightly towards the fuel cap on the left hand side, they are closer to the hole on the left than the right. Otherwise, it looks like all of it was fixed. Once I go back to master, I will make sure the first flickering thing was fixed, but otherwise, apologies for wasting your time ;)

**Comment by wlbragg** [comment]
>rivets on the left seem pinched slightly towards the fuel cap on the left hand side, they are closer to the hole on the left than the right

Yeah, like @gilbertohasnofb said, this stuff is after the fact and that makes it really difficult to work with. We probable should let it go. The mapping of the wings was a nightmare for me, I'd probably do more harm than good tying to correct that.
The error in the first image you posted was worth correcting though (which we did of course).

**Comment by wlbragg** [comment]
Oops, hit the wrong button!

**Comment by gilbertohasnofb** [comment]
I agree, I don't think this is worth the trouble. IMO we can close this.
